tendenci/apps/events/models.py

```diff
@@ -174,6 +174,9 @@ class Place(models.Model):
     country = models.CharField(max_length=150, blank=True)
     national = models.BooleanField(default=False, help_text=_('Is it a national event?'))
 
+    use_zoom_integration = models.BooleanField(default=False)
```

Comment:
These two fields, `use_zoom_integration` and `is_zoom_webinar`, are duplicates in the Place model.